src/hub/InterfacesCard.tsx

```diff
@@ -0,0 +1,86 @@
+import { Card, Col, Row, Statistic } from "antd";
+import React from "react";
+import CenteredSpinner from "../layout/CenteredSpinner";
+import ErrorAlert from "../layout/ErrorAlert";
+import { InfoCircleOutlined, PlusCircleOutlined } from "@ant-design/icons";
+import { Link } from "react-router-dom";
+import { InterfaceRevisionWithKey } from "./Interfaces.container";
+
+interface InterfacesCardProps {
+  interfaces?: InterfaceRevisionWithKey[];
+  error?: Error;
+  isLoading: boolean;
+}
+
+function InterfacesCard({ interfaces, error, isLoading }: InterfacesCardProps) {
+  if (isLoading) {
+    return <CenteredSpinner />;
+  }
+
+  if (error) {
+    return <ErrorAlert error={error} />;
+  }
+
+  const cardActions = (rev: InterfaceRevisionWithKey) => {
+    const btns = [
+      <Link to={`/actions/new/${rev?.metadata.path}/${rev?.revision}`}>
+        <PlusCircleOutlined /> Create
+      </Link>,
+    ];
+    if (rev?.metadata.documentationURL) {
+      btns.unshift(
+        <a
+          href={rev.metadata.documentationURL}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <InfoCircleOutlined /> Action Doc
```

Comment:
Maybe Create Action and Documentation will be more clear?

```suggestion
        <PlusCircleOutlined /> Create Action
      </Link>,
    ];
    if (rev?.metadata.documentationURL) {
      btns.unshift(
        <a
          href={rev.metadata.documentationURL}
          target="_blank"
          rel="noopener noreferrer"
        >
          <InfoCircleOutlined /> Documentation
```